src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1180,23 +1197,23 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
                                         $indent = 4
 
-                                        $errorCategoryMsg = & { Set-StrictMode -Version 1; $_.ErrorCategory_Message }
+                                        $errorCategoryMsg = $err.ErrorCategory_Message
 
                                         if ($null -ne $errorCategoryMsg)
                                         {
-                                            $indentString = '+ CategoryInfo          : ' + $_.ErrorCategory_Message
+                                            $indentString = '+ CategoryInfo          : ' + $err.ErrorCategory_Message
```

Comment:
hardcoded tab?

Reply:
This is the existing NormalView code for the spaces.  Yeah, it's hardcoded to match FullyQualifiedErrorId spacing.